src/OpenTelemetry.Exporter.Geneva/GenevaLoggingExtensions.cs

```diff
@@ -27,4 +30,102 @@ public static OpenTelemetryLoggerOptions AddGenevaLogExporter(this OpenTelemetry
             return options.AddProcessor(new ReentrantExportProcessor<LogRecord>(exporter));
         }
     }
+
+    /// <summary>
+    /// Adds Geneva exporter to the LoggerProvider.
+    /// </summary>
+    /// <param name="builder"><see cref="LoggerProviderBuilder"/> builder to use.</param>
+    /// <returns>The instance of <see cref="LoggerProviderBuilder"/> to chain the calls.</returns>
+    public static LoggerProviderBuilder AddGenevaLogExporter(this LoggerProviderBuilder builder)
+        => AddGenevaLogExporter(builder, name: null, configureExporter: null);
+
+    /// <summary>
+    /// Adds Geneva exporter to the LoggerProvider.
+    /// </summary>
+    /// <param name="builder"><see cref="LoggerProviderBuilder"/> builder to use.</param>
+    /// <param name="configureExporter">Callback action for configuring <see cref="GenevaExporterOptions"/>.</param>
+    /// <returns>The instance of <see cref="LoggerProviderBuilder"/> to chain the calls.</returns>
+    public static LoggerProviderBuilder AddGenevaLogExporter(this LoggerProviderBuilder builder, Action<GenevaExporterOptions> configureExporter)
+        => AddGenevaLogExporter(builder, name: null, configureExporter);
+
+    /// <summary>
+    /// Adds Geneva exporter to the LoggerProvider.
+    /// </summary>
+    /// <param name="builder"><see cref="LoggerProviderBuilder"/> builder to use.</param>
+    /// <param name="name">Optional name which is used when retrieving options.</param>
+    /// <param name="configureExporter">Optional callback action for configuring <see cref="GenevaExporterOptions"/>.</param>
+    /// <returns>The instance of <see cref="LoggerProviderBuilder"/> to chain the calls.</returns>
+    public static LoggerProviderBuilder AddGenevaLogExporter(
+        this LoggerProviderBuilder builder,
+        string name,
+        Action<GenevaExporterOptions> configureExporter)
```

Comment:
nit:

Consider adding `#nullable enable` to this file so you can decorate these...

```csharp
    public static LoggerProviderBuilder AddGenevaLogExporter(
        this LoggerProviderBuilder builder,
        string? name,
        Action<GenevaExporterOptions>? configureExporter)
```

I think the nullable decoration makes it more clear what the optional things are. But it doesn't look like tracing or metrics are decorated either so up to you if you want to take that on separately or not at all 😄

Reply:
If possible, you could enable nullable in whole project in separate PR.
Now, only 5 (Geneva, Instana, ElasticsearchClient, GrpcCore, and HTTP) _production_ projects are missing this functionality. Please include also testing projects to such change.

Reply:
I will do it as a separate item.